GEOSldas_App/ldas_setup

```diff
@@ -153,6 +154,9 @@ class LDASsetup:
 
         for key in rqdExeInpKeys :
             assert key in self.rqdExeInp,' "%s" is required in the input file %s' % (key,self.exeinpfile)
+        if cmdLineArgs['nymdb' ] != 'None' :
+            self.date  = f"{self.nymdb} 210000"
```

Comment:
I think we will need to pass in HHMMSS as well, or at least HH.  Should this ever run in FP, we cannot hardwire 21z.

Reply:
will add HHMMSS.